jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,6 +1,9 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: keyword_filter.schema.yaml
+- $ref: keyword_exptype.schema.yaml
```

Comment:
So I say go ahead and merge this fairly soon, so that the basics (i.e. the new referencefile.schema.yaml and a few of the keyword_xxxx.schema.yaml files) are in there and available for everyone to start building on.